x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/index.d.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { Size } from '../../../../../types';
+
+export interface CaptureConfig {
+    zoom: number;
+    viewport: Size;
+  }
+  
+export interface ViewZoomWidthHeight {
+    zoom: number;
+    width: number;
+    height: number;
+  }
```

Comment:
same comment with this file and the space aligning.  Looks like we'll have to update `.d.ts` files automatically.  This will probably get hit by the linter when ci runs.